static/src/javascripts/enhanced-core.js

```diff
@@ -1,7 +1,5 @@
 define([
-    // 3rd party libs
-    'raven',
-    'picturefill',
+    // The following modules are here for bundling purposes only
```

Comment:
maybe expand this comment to mention that it only includes internal modules? just a way of preventing individual bootstraps duplicating modules (is that correct?)